src/Bridge/Symfony/DependencyInjection/FidryAliceDataFixturesExtension.php

```diff
@@ -44,12 +41,12 @@ public function load(array $configs, ContainerBuilder $container)
         $processedConfiguration = $this->processConfiguration($configuration, $configs);
         $bundles = array_flip($container->getParameter('kernel.bundles'));
 
-        if (false === array_key_exists(self::NELMIO_ALICE_BUNDLE_CLASS, $bundles)) {
+        if (false === array_key_exists(NelmioAliceBundle::class, $bundles)) {
```

Comment:
doesn't this break tests? If the bundle is not installed then this check fails which defeat its point, hence the usage of constants here